spec/models/recipes_spec.rb

```diff
@@ -61,9 +65,10 @@
   describe '.with_text' do
     subject { described_class.with_text(query) }
 
+    let(:user) { create(:user) }
     let!(:recipes) do
```

Comment:
Factory bot has a nice way of creating multiple records
`let!(:recipes) { create_list(:recipe, 3) }`